Cork/Views/Sidebar/Components/Casks Section.swift

```diff
@@ -9,100 +9,38 @@ import SwiftUI
 
 struct CasksSection: View {
     
-    @AppStorage("allowMoreCompleteUninstallations") var allowMoreCompleteUninstallations: Bool = false
-    
     @EnvironmentObject var appState: AppState
     @EnvironmentObject var brewData: BrewDataStorage
-    @EnvironmentObject var outdatedPackageTracker: OutdatedPackageTracker
     
-    @Binding var searchText: String
+    let searchText: String
```

Comment:
All 3 section views had `searchText` as a two-way binding, but never modified it